### DISCUSSION TITLE
Layout shifts to bottom when error message is added

### DISCUSSION THREAD

**Discussion by mehrdad-shokri** [mantinedev/mantine#6658] [Q&A]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

^7.12.0

### What package has an issue?

@mantine/core

### What framework do you use?

Next.js

### In which browsers you can reproduce the issue?

Chrome

### Describe the bug

When Adding error to text field, the elements underneath gets shifted to bottom.
[file.webm](https://github.com/user-attachments/assets/ae50325b-62f5-4422-871f-d1a23d0c55fe)

### Possible fix

I think one candidate is passing a prop to reserve the height of the error field, so it doesn't get shifted when it's added to the element

### Self-service

- [X] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment]
If you want error message to not take space, give it absolute position with Styles API.

**Reply by mehrdad-shokri** [reply]
Hello, is it possible to error message take the space always including the times there is no error message?

**Reply by rtivital** [reply]
Mantine does not provide such feature

**Reply by mehrdad-shokri** [reply]
Do you agree this is a ux issue? and it's present to all Mantine apps.

**Comment by chumba-wamba** [comment]
What is the correct way of achieving the expected behaviour (aka no layout shift) in 2025? Is it still by providing `position: absolute` to the error message?
